packages/components/ng-ovh-payment-method/src/components/choice/template.html

```diff
@@ -11,7 +11,7 @@
     <oui-radio
         data-name="paymentMethod"
         data-model="$ctrl.model"
-        data-value="true"
+        data-value="$ctrl.defaultPaymentMethod"
```

Comment:
I think no need to use data- for name, value, disabled and required as they are standard HTML attributes